### PR TITLE
New: Improved Maintenance Release Wording

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
title
#### Todos
- Tests
- Wiki Updates

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: frontend/src/App/AppUpdatedModalContent.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ function AppUpdatedModalContent(props) {
             <div>
               {
                 !update.changes &&
-                  <div className={styles.maintenance}>Maintenance release</div>
+                  <div className={styles.maintenance}>Maintenance Release: bug fixes and other improvements. See Github Commit History for more details</div>
```
Comment (Stanzilla) [inline frontend/src/App/AppUpdatedModalContent.js:79]:
```suggestion
                  <div className={styles.maintenance}>Maintenance Release: Bug fixes and other improvements. See <a href="https://github.com/Sonarr/Sonarr" title="GitHub Project" target="_blank" rel="noopener">GitHub Commit History</a> for more details.</div>
```

Maybe?

Comment (markus101) [reply frontend/src/App/AppUpdatedModalContent.js:79]: A link to GitHub commits seems reasonable, but should use Link.

Bug fixes should not be part of a build that only contains minor code changes, I think something like:

> Maintenance Release: See <GitHub Commits> for the technical details

File: frontend/src/App/AppUpdatedModalContent.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ function AppUpdatedModalContent(props) {
             <div>
               {
                 !update.changes &&
-                  <div className={styles.maintenance}>Maintenance release</div>
+                  <div className={styles.maintenance}>Maintenance Release: See Github Commit History for the technical details.</div>
```
Comment (markus101) [inline frontend/src/App/AppUpdatedModalContent.js:79]:
```suggestion
                  <div className={styles.maintenance}>Maintenance Release: See GitHub commit history for details.</div>
```

This corrects capitalization and in retrospect we don't need technical details. A link directly to the commits could be useful though.

**Review by markus101 — CHANGES_REQUESTED** [review]

File: frontend/src/System/Updates/Updates.js [hunk 3]
```diff
@@ -181,7 +181,7 @@ class Updates extends Component {
 
                         {
                           !hasChanges &&
-                            <div>Maintenance release</div>
+                            <div>Maintenance Release: See Github Commit History for the technical details.</div>
```
Comment (markus101) [inline frontend/src/System/Updates/Updates.js:184]: Same change is needed here

**Pull request merged**